### PR TITLE
feat: revamp home and add copy feature

### PR DESCRIPTION
**Pull request by cabreraalex**
Fix EOT-46

### CODE REVIEW

File: src/lib/components/home/HomeRow.svelte [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
```
Comment (Sparkier) [inline src/lib/components/home/HomeRow.svelte:1]: Naming: This is not really a row, right? It's just one element in the row.

**Review by Sparkier — APPROVED** [review]
I made a very small change to the triple dot menu to make it a bit more subtle and playful so take a look. Other than the name for the component really like it.

File: src/routes/api/project/[id]/copy/+server.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		oldInstances.map((instance) => ({
+			project_id: newProject.id,
+			input: instance.input,
+			label: instance.label
+		}))
+	);
+
+	if (newInstancesRes.error) {
+		error(500, 'Error creating new instances');
+	}
```
[the remark this answers is not in the document]
Comment (cabreraalex) [reply src/routes/api/project/[id]/copy/+server.ts:63]: yeah not great, linear issue sgtm

File: src/lib/components/home/HomeRow.svelte (hunk 1 above)
Comment (cabreraalex) [reply src/lib/components/home/HomeRow.svelte:1]: yup good catch will rename

File: src/routes/api/project/[id]/copy/+server.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		.eq('project_id', projectId)
+		.order('created_at', { ascending: false })
+		.limit(1);
+	const oldInstancesReq = supabase.from('instances').select('*').eq('project_id', projectId);
+
+	const [oldProjectRes, oldPromptRes, oldInstancesRes] = await Promise.all([
+		oldProjectReq,
+		oldPromptReq,
+		oldInstancesReq
+	]);
```
[the remark this answers is not in the document]
Comment (cabreraalex) [reply src/routes/api/project/[id]/copy/+server.ts:24]: It got complicated to find the right new instance id for each prediction, got lazy. Suggestions we could copy... idk

[the remark this answers is not in the document]
Comment (Sparkier) [reply src/routes/api/project/[id]/copy/+server.ts:24]: if complicated don't worry lets regenerate

**Pull request merged**